include/posts/class-bulk-delete-posts.php

```diff
@@ -212,12 +212,18 @@ public static function render_delete_posts_by_category_box() {
         <table class="form-table">
             <tr>
                 <td scope="row">
-				<select class="select2" name="smbd_cats[]" multiple data-placeholder="<?php _e( 'Select Categories', 'bulk-delete' ); ?>">
-					<option value="all"><?php _e( 'All Categories', 'bulk-delete' ); ?></option>
-		<?php foreach ( $categories as $category ) { ?>
-			<option value="<?php echo $category->cat_ID; ?>"><?php echo $category->cat_name, ' (', $category->count, ' ', __( 'Posts', 'bulk-delete' ), ')'; ?></option>
-		<?php } ?>
+                <?php if( count($categories) > 50 ){?>
+					<select class="select2Ajax" name="smbd_cats[]" data-term="category" multiple data-placeholder="<?php _e( 'Select Categories', 'bulk-delete' ); ?>" style="width:300px">
```

Comment:
The attribute `data-term` should be `data-taxonomy`. This is applicable for tags as well.